oauth2/src/main/java/com/okta/spring/boot/oauth/env/OktaOAuth2PropertiesMappingEnvironmentPostProcessor.java

```diff
@@ -181,7 +181,7 @@ private ConditionalMapPropertySource(String name, Map<String, Object> source, En
         @Override
         public Object getProperty(String name) {
             if (OKTA_ISSUER_URI.equals(name) && OKTA_STATIC_DISCOVERY.equals(this.getName()) && !containsProperty(name)) {
-                LOGGER.warn("Mandatory property `" + OKTA_OAUTH_ISSUER + "` is missing");
+                log.warn("Mandatory property `" + OKTA_OAUTH_ISSUER + "` is missing");
```

Comment:
I mentioned this earlier in the thread, but use something like this: https://github.com/okta/okta-commons-java/blob/master/config-check/src/main/java/com/okta/commons/configcheck/ConfigurationValidator.java#L176

The warning should tell the user what they need to do to fix the problem

Reply:
`validateIssuer` method is already used in the OktaOAuth2Properties class. I don't see the need to perform the same check twice, if you mean to use it here.
But when the property is missing - application will fall much earlier, long before validations from OktaOAuth2Properties class.